Predictor_submission_template.ipynb

```diff
@@ -369,15 +369,160 @@
+    "- Intervention Plans are the official ones"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "validate(start_date=\"2020-08-01\",\n",
+    "         end_date=\"2020-08-04\",\n",
+    "         ip_file=\"../../../validation/data/2020-09-30_historical_ip.csv\",\n",
+    "         output_file=\"predictions/val_4_days.csv\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1 month in the future\n",
+    "- 2 countries only\n",
+    "- there's a gap between date of last known number of cases and start_date\n",
+    "- For future dates, Intervention Plans contains scenarios for which predictions are requested to answer the question: what will happen if we apply these plans?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "validate(start_date=\"2021-01-01\",\n",
+    "         end_date=\"2021-01-31\",\n",
+    "         ip_file=\"../../../validation/data/future_ip.csv\",\n",
+    "         output_file=\"predictions/val_1_month_future.csv\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 180 days, from a future date, all countries and regions\n",
+    "- Prediction start date is 1 week from now. (i.e. assuming submission date is 1 week from now)  \n",
+    "- Prediction end date is 6 months after start date.  \n",
+    "- Prediction is requested for all available countries and regions.  \n",
+    "- Intervention plan scenario: freeze last known intervention plans for each country and region.  \n",
+    "\n",
+    "As the number of cases is not known yet between today and start date, but the model relies on them, the model has to predict them in order to use them.  \n",
+    "This test is the most demanding test. It should take less than 1 hour to generate the prediction file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Generate the scenario"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datetime import datetime, timedelta\n",
+    "\n",
+    "start_date = datetime.now() + timedelta(days=7)\n",
+    "start_date_str = start_date.strftime('%Y-%m-%d')\n",
+    "end_date = start_date + timedelta(days=180)\n",
+    "end_date_str = end_date.strftime('%Y-%m-%d')\n",
+    "print(f\"Start date: {start_date_str}\")\n",
+    "print(f\"End date: {end_date_str}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from validation.scenario_generator import get_raw_data, generate_scenario, NPI_COLUMNS\n",
+    "DATA_URL = 'https://raw.githubusercontent.com/OxCGRT/covid-policy-tracker/master/data/OxCGRT_latest.csv'\n",
+    "latest_df = get_raw_data(DATA_FILE, latest=True)\n",
```

Comment:
`DATA_FILE` vs `DATA_URL`?

Reply:
Either works here, but I preferred to use the latest data instead of some potentially stale cached data